geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateDiskStoreCommand.java

```diff
@@ -154,6 +183,32 @@ private DiskStoreType createDiskStoreType(String name, DiskStoreAttributes diskS
     return diskStoreType;
   }
 
+  @SuppressWarnings("unchecked")
+  List<DiskStoreDetails> getDiskStoreListing(Set<DistributedMember> members) {
```

Comment:
if we can assume that cluster configuration is the single source of truth, we can just find out the existing diskstores in the cluster config.

Reply:
but then again, there are cases when cluster configuration service is not enabled....